src/app/home/about/about.component.scss

```diff
@@ -0,0 +1,21 @@
+#layer {
+  background-color: rgba(100, 181, 246, 0.45);
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  z-index: 1;
+}
+.title-about-color {
+  color: #64B5F6;
+  font-size: 20pt;
```

Comment:
Try to use standard font sizes, we are not defining this kind of styles. Try checking other components.